frontend/src/bundles/talent-onboarding/components/contacts-cv-step/contacts-cv-step.tsx

```diff
@@ -339,7 +345,9 @@ const ContactsCVStep: React.FC = () => {
                     />
 
                     <FormHelperText className={styles.fileError}>
-                        {errors.cv?.type === 'fileSize' && errors.cv.message}
+                        {errors.cv?.type === 'fileSize' ||
+                            (errors.cv?.type === 'fileType' &&
+                                errors.cv.message)}
```

Comment:
There is a typo with brackets, I don`t see an error when trying to upload file greater than 5mb
{(errors.cv?.type === 'fileSize' || errors.cv?.type === 'fileType') && errors.cv.message}

Reply:
fixed